roles/uperf/templates/server.yml.j2

```diff
@@ -1,47 +1,65 @@
 ---
-kind: Job
-apiVersion: batch/v1
-metadata:
-  name: 'uperf-server-{{ item }}-{{ trunc_uuid }}'
-  namespace: "{{ operator_namespace }}"
-spec:
-  ttlSecondsAfterFinished: 600
-  backoffLimit: 0
-  template:
-    metadata:
-      labels:
-        app: uperf-bench-server-{{item}}-{{ trunc_uuid }}
-        type: uperf-bench-server-{{ trunc_uuid }}
+apiVersion: v1
+kind: List
+metadata: {}
+items:
+{% macro job_template(item, node_idx_item='') %}
+  - kind: Job
+    apiVersion: batch/v1
+    metadata:    
+      name: 'uperf-server-{{worker_node_list[ node_idx_item ] }}-{{ item }}-{{ trunc_uuid }}'
+      namespace: "{{ operator_namespace }}"
+    spec:
+      ttlSecondsAfterFinished: 600
+      backoffLimit: 0
+      template:
+        metadata:
+          labels:
+            app: uperf-bench-server-{{ worker_node_list[node_idx_item] | truncate(16,true,'') }}-{{ item  }}-{{ trunc_uuid }}
```

Comment:
s/bench// 

I wonder if we can do something like:  `worker_node_list[node_idx_item].split('.')[0]` 

Which would get us something like `uperf-server-ip-10-0-133-30`

Reply:
I intentionally put node name in there to identify them easier. Truncating the long node name still leaves good context.